action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -185,10 +186,18 @@ func (stateDB *StateDBAdapter) Exist(evmAddr common.Address) bool {
 	return true
 }
 
-// Empty empties the contract
-func (stateDB *StateDBAdapter) Empty(common.Address) bool {
-	logger.Debug().Msg("Empty is not implemented")
-	return false
+// Empty returns true if the the contract is empty
+func (stateDB *StateDBAdapter) Empty(evmAddr common.Address) bool {
+	addr := address.New(stateDB.cm.ChainID(), evmAddr.Bytes())
+	logger.Debug().Msgf("Check whether the contract is empty")
+	state, err := stateDB.AccountState(addr.IotxAddress())
+	if err != nil || state == nil {
```

Comment:
use state == state.EmptyAccount
there's a change recently, if account does not exist, will return state.EmptyAccount

Reply:
And error of reading state DB should be treated differently